scripts/performance/report-as-a-pr-comment.js

```diff
@@ -0,0 +1,225 @@
+const { command } = require('../lib/command')
+const { fetchHandlingError } = require('../lib/execution-utils')
+const { getOrg2ApiKey, getOrg2AppKey } = require('../lib/secrets')
+const { LOCAL_BRANCH, BASE_BRANCH, GITHUB_TOKEN, getLastCommonCommit, fetchPR } = require('../lib/git-utils')
+const PR_COMMENT_HEADER = 'Bundles Sizes Evolution'
+const PR_COMMENTER_AUTH_TOKEN = command`authanywhere`.run().split(' ')[2].trim()
+const ONE_DAY_IN_SECOND = 24 * 60 * 60
+// The value is set to 5% as it's around 10 times the average value for small PRs.
+const SIZE_INCREASE_THRESHOLD = 5
+const LOCAL_COMMIT_SHA = process.env.CI_COMMIT_SHORT_SHA
+const ACTION_NAMES = [
+  'adderror',
+  'addaction',
+  'logmessage',
+  'startview',
+  'startstopsessionreplayrecording',
+  'addtiming',
+  'addglobalcontext',
+]
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about extracting all the logic related to the performance data from the reporting logic?
it could be easier to follow from main if we had something like
```js
main() {
  const bundleSizes = calculateBundleSizes()
  const performanceData = await computePerformanceData() // run synthetics test under the hood
  await reportToDatadog(bundleSizes)
  await reportAsPrComment(bundleSizes, performanceData)
}
```